src/views/GovernanceReviewTeam/Summary/index.tsx

```diff
@@ -206,13 +214,12 @@ const RequestSummary = ({
             <Grid desktop={{ col: 8 }}>
               <div>
                 <h4 className="margin-right-1">{t('status.label')}</h4>
-                <StateTag state={state} />
+                <StateTag state={flags.itGovV2Fields ? state : stateV1} />
               </div>
               <p className="text-base-dark" data-testid="grt-current-status">
-                {
-                  /* TODO EASI-3440: Update to use v2 statuses */
-                  translateStatus(status, lcid)
-                }
+                {flags.itGovV2Fields
```

Comment:
I know the AC said otherwise, but this close to launch we might just want to modify references to the current flag (`flags.itGovV2Enabled`)